pkgs/applications/system/zxfer/default.nix

```diff
@@ -0,0 +1,45 @@
+{ coreutils
+, fetchFromGitHub
+, gawk
+, gnugrep
+, gnused
+, lib
```

Comment:
There's a convention for putting `lib` first on the list, then going in the order they're used.